api/Window.json

```diff
@@ -1585,10 +1585,10 @@
               "version_added": "24"
             },
             "safari": {
-              "version_added": "6.1"
+              "version_added": "3"
```

Comment:
This does not seem plausible, https://w3c.github.io/webcrypto/ is too new for this. http://mdn-bcd-collector.appspot.com/tests/api/Window/crypto does show true in Safari 5, but this seems like a flaw in our tooling. Can you check it out?

I'll hold off reviewing the rest, since there are more 3s added in this PR, and now I distrust them all until this is explained :)

Reply:
Alright, so I dug deeper into this and found out what's going on.  Apparently, the `crypto` variable _is_ in the `window` instance in Safari 3, but it is set to `undefined`.  I guess we'll also need to configure our tests to check if the variable is not `undefined`...

<img width="594" alt="Screen Shot 2020-10-08 at 09 16 13" src="https://user-images.githubusercontent.com/5179191/95485664-eb00f000-0946-11eb-9eea-933d8e21dab3.png">

Reply:
Also discussed in https://github.com/mdn/browser-compat-data/pull/6838#discussion_r501844503